packages/ml_api/docker/.env.sample

```diff
@@ -0,0 +1,5 @@
+DB_USER=gvenik
+DB_PASSWORD=password
+DB_PORT=7617
+DB_HOST=0.0.0.0
+DB_NAME=just_a_db
```

Comment:
Let's change 'gvenik' to 'user ' :), i.e. `DB_USER = user`. Also, let `DB_NAME = my_db`.